app/controllers/catalog_controller.rb

```diff
@@ -2,7 +2,6 @@ class CatalogController < ApplicationController
   include Hydra::Catalog
   include Hydra::Controller::ControllerBehavior
   include Sufia::Catalog
-  include BlacklightAdvancedSearch::Controller
 
```

Comment:
are you intentionally removing AdvancedSearch-like behavior here?  Is it related to the Sufia upgrade?